aio/tests/e2e/src/pwa-manifest.po.ts

```diff
@@ -0,0 +1,74 @@
+import { get as httpGet } from 'http';
+import { get as httpsGet } from 'https';
+import { browser } from 'protractor';
+import { SitePage } from './app.po';
+
+
+export type Json = null | boolean | number | string | Json[] | { [key: string]: Json };
+
+/**
+ * The shape of a PWA manifest.
+ * For simplicity, we only define types for the properties we care about in tests.
+ * @see https://developer.mozilla.org/en-US/docs/Web/Manifest
+ */
+export type PwaManifest = Json & {
+  shortcuts?: PwaShortcutItem[];
+};
```

Comment:
I believe the norm is that if you are describing a type literal then the delimiter is commas:

```suggestion
export type PwaManifest = Json & {
  shortcuts?: PwaShortcutItem[],
};
```

(and similarly for `PwaShortcutItem` below).

Reply:
Done.

Reply:
I turns out that there is no "norm" - not even in our own codebase here. So I am happy for you to stick with semi-colons as you wish.

Reply:
Too late 😛